pcdsdevices/signal.py

```diff
@@ -1014,6 +1013,14 @@ def kind(self):
     def kind(self, value):
         self._epics_signal.kind = value
 
+    def __eq__(self, other):
+        if isinstance(other, EpicsSignal) and self.should_use_epics_signal:
+            return self == other.parent
+        return NotImplemented
```

Comment:
Do we want to return NotImplemented here?  Or raise it?

In fact, do we ever want this method to raise an error?  Shouldn't we just return `False` in those cases?